release_notes/topics/4_0_0_final.adoc

```diff
@@ -1,3 +1,15 @@
-= Test Feature
+= Client Scopes and suport for OAuth 2 scope parameter
 
-Some test feature.
\ No newline at end of file
+We added support for Client Scopes. Client Scopes replaced Client Templates. They allow more flexibility and better support for `scope` parameter,
```

Comment:
We added support for Client Scopes, which replaces Client Templates. Client Scopes are a more flexible approach and also provides better support for the OAuth `scope` parameter.